### DISCUSSION TITLE
Z1-mini Zigbee development board

### DISCUSSION THREAD

**Discussion by echopage** [indomus/forum#28] [Hardware & Firmware]
Qualcuno lo ha già provato? 
https://gio-dot.github.io/Z1-Mini/

**Comment by gio-dot** [comment]
Ciao, sono l'autore del progetto, se vuoi chiedi pure in attesa di altri commenti.

**Comment by echopage** [comment]
Ciao, prima di tutto complimenti per il progetto, ho letto della tua board sulla community di hassio, leggendo ho scoperto che stai lavorando sull'implementazione di alimentazione a batteria a che punto sei? Penso che il progetto sia interessantissimo, ma che il nodo cruciale sia la possibilità di utilizzarlo, per alcuni progetti,  con normali pile per poterlo svincolare dalla rete elettrica come la maggior parte, p.e. dei sensori xiaomi!!!!

**Reply by gio-dot** [reply]
La parte firmware non è sviluppata da me come avrai visto. Io mi sono impegnato nel creare un hardware facile da usare e ben documentato con esempi etc. (altri lavori arriveranno, sto disegnando anche delle shield apposite).
Detto questo il firmware PTVO esiste con implementata la gestione dello sleep nella sua versione a pagamento. Ci sono poi altri firmware liberi che permettono di lavorare a batteria, come ad esempio:
https://github.com/diyruz/freepad
https://diyruz.github.io/posts/reed/
